packages/at_secondary_server/lib/src/verb/handler/keys_verb_handler.dart

```diff
@@ -0,0 +1,195 @@
+            // for now skip commit log for keys created through this verb.
+            // If we update/create other keys in the future e.g publickey@alice, @bob:shared_key@alice, change the impl
+            result = await keyStore.put(keyName, atData, skipCommit: true);
+          }
+          response.data = result.toString();
+        } catch (e, st) {
+          logger.warning('$e\n$st');
+        }
+        break;
+      case 'get':
+        final keyVisibility = verbParams[visibility];
+        final keyNameFromParams = verbParams[keyName];
+        logger.finer('keyVisibility: $keyVisibility');
+        logger.finer('keyNameFromParams: $keyNameFromParams');
+        List<dynamic> result;
+        final filteredKeys = [];
+        if (keyNameFromParams != null && keyNameFromParams.isNotEmpty) {
+          try {
+            final value = await keyStore.get(keyNameFromParams);
+            response.data = value.data;
+            break;
+          } on KeyNotFoundException {
+            throw KeyNotFoundException(
+                'key $keyNameFromParams not found in keystore');
+          }
+        }
+        logger.finer('hasManageAccess: $hasManageAccess');
+        if (keyVisibility != null && keyVisibility.isNotEmpty) {
+          if (hasManageAccess) {
+            result = keyStore.getKeys(
+                regex:
+                    '.*$keyVisibility.*__global$atSign\$|.*$keyVisibility.*__manage$atSign\$');
+          } else {
+            result = keyStore.getKeys(
+                regex: '.*__${keyVisibility}_keys.__global$atSign\$');
+          }
+          for (String key in result) {
+            await _addKeyIfEnrollmentIdMatches(
+                filteredKeys, key, enrollIdFromMetadata!);
+          }
+          var keyMap = {
+            'private':
+                '$enrollIdFromMetadata.$defaultEncryptionPrivateKey.$enrollManageNamespace$atSign',
+            'self':
+                '$enrollIdFromMetadata.$defaultSelfEncryptionKey.$enrollManageNamespace$atSign',
+          };
+
+          var keyString = keyMap[keyVisibility];
+          if (keyString != null) {
+            dynamic value;
+            try {
+              value = await keyStore.get(keyString);
+            } on KeyNotFoundException {
+              logger.warning('key $keyString not found');
+            }
+            if (value != null && value.data != null) {
+              filteredKeys.add(keyString);
+            }
+          }
+
+          response.data = jsonEncode(filteredKeys);
+        }
+        break;
+      case 'delete':
+        final keyNameFromParams = verbParams[keyName];
+        logger.finer('keyNameFromParams: $keyNameFromParams');
+        response.data =
+            (await keyStore.remove(keyNameFromParams, skipCommit: true))
+                .toString();
+        break;
+    }
+  }
+
+  // Function to add a key to filteredKeys if enrollmentId matches
+  Future<void> _addKeyIfEnrollmentIdMatches(List<dynamic> filteredKeys,
```

Comment:
The method comment is almost literally just repeating the name of the method. Can you make it more helpful?

Also - general point - please use dart docs syntax - class / method / function / variable comments use `///` not `//`.

Reply:
Done.
https://github.com/atsign-foundation/at_server/pull/1496/commits/b22ea2028164aa54299758080d3b9033bf3a98e2